### PR TITLE
Doc: add cleanup instructions for docker-compose scenario

### PR DESCRIPTION
**Pull request by bzz**
Solves #210

### CODE REVIEW

**Comment by carlosms** [comment]
If I'm not mistaken this command would only apply if we started the containers with `up -d`, which is not the case.

Following the instructions from the readme the `up` command stays in the foreground and Ctrl+C is enough to stop it, which should be obvious to anyone familiar with the shell.

**Comment by carlosms** [comment]
Oh, and I've just realized that we do indeed already mention it:
> To kill the running containers use `Ctrl+C`.

**Comment by bzz** [comment]
@carlosms I would keep it as it's useful in case:

 - using https://github.com/src-d/gitbase-playground#with-docker-compose
 - user runs `docker-compose up`
 - but there already is bblfsd container on his machine on the same port
 - so the process exists, leaving some containers running, some not (lookout and bblfsh shutdown, analyzer and db still running)

So reminder of using `docker-compose down` to clean things up may be nice to have.

**Comment by carlosms** [comment]
I see your point, but at the same time from my point of view that's troubleshooting and not normal usage, and it is written as a required step after every usage.

I just think it's not needed, but I'm not strongly against it. Please go ahead and merge if you consider it's useful.

File: README.md [hunk 1]
```diff
@@ -33,6 +33,12 @@ $ GITBASEPG_REPOS_FOLDER=$HOME/repos docker-compose up --force-recreate
 
 The server should be now available at [http://localhost:8080](http://localhost:8080).
 
+In case if there are any containers left, you can use
```
Comment (carlosms) [inline README.md]: I believe this needs to be **In case there are** or **If there are**

**Comment by bzz** [comment]
Thank you @carlosms ! History re-written

**Pull request merged**